torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -233,6 +233,12 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
   } else if (node->kind() == onnx::Mul) {
     updated_val = at::mul(inputTensorValues[0], inputTensorValues[1]);
     return c10::optional<at::Tensor>(updated_val);
+  } else if (node->kind() == onnx::Sub) {
```

Comment:
A potential more generic approach is to abstract the Mul/Div/Sub/Add as binary op,  the pseudo code may be as following: 

if (node->kind() == onnx::Div || node->kind() == onnx::Mul ...) {
     kind2func k2f = { {onnx::Mul: at::mul}, ... };
     function f = k2f(node->kind());
     return c10::optional<at::Tensor>(f(inputTensorValues[0], inputTensorValues[1]));
}

Reply:
After discussion with Tianyou,  I take back my comments since aten::add/sub has different argument number against aten::div/mul ( 3 against 2), although we may use bind tricks to unify the interface, it is a little bit overkill. I think Tianyou's current implementation is good enough.